pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -709,6 +698,19 @@ protected void internalDeletePartitionedTopic(AsyncResponse asyncResponse, boole
                 log.error("[{}] Failed to delete partitioned topic {}", clientAppId(), topicName, e1);
                 asyncResponse.resume(new RestException(e1));
             }
+        }).exceptionally(ex ->{
+            Throwable cause = ex.getCause();
```

Comment:
This ex could be `new RestException(Status.FORBIDDEN, "Need to authenticate to perform the request")`

Reply:
fixed,